libs/application/templates/health-insurance/src/forms/messages.ts

```diff
@@ -437,22 +437,27 @@ export const m = defineMessages({
     defaultMessage: 'Leiðbeiningar',
     description: 'How to register',
   },
-  activeApplicationTitle: {
-    id: 'hi.application:activeApplication.title',
-    defaultMessage: 'Virk umsókn',
-    description: 'Active application',
+  activeDraftApplicationTitle: {
+    id: 'hi.application:activeDraftApplication.title',
+    defaultMessage: 'Open your previous draft',
+    description: 'Open your previous draft',
   },
-  activeApplicationDescription: {
-    id: 'hi.application:activeApplication.description',
+  activeDraftApplicationDescription: {
+    id: 'hi.application:activeDraftApplication.description',
     defaultMessage:
-      'Þú hefur nú þegar sótt um sjúkratryggingu. Við umsókn voru sendar upplýsingar á netfang sem gefið var upp. Þú getur alltaf séð stöðu umsóknar á Mínum síðum',
+      'You have already started to fill out an application. Please open your previous draft to continue.',
     description:
-      'You have already submitted an application for health insurance. We will notify you on the e-mail address you provided in the application when the status changes. You can always see your application status in My Pages.',
+      'You have already started to fill out an application. Please open your previous draft to continue.',
```

Comment:
Just passing a comment I received on my PR: We should keep the description field as description and not use the full English translation. But not a big deal to update all descriptions, just to keep in mind when we add/update this file :)

Reply:
Okay, will change and keep in mind for future changes.